subsys/bluetooth/Kconfig.logging

```diff
@@ -428,6 +428,14 @@ config BT_DEBUG_HFP_HF
 	  This option enables debug support for the Bluetooth
 	  Hands Free Profile (HFP).
 
+config BT_DEBUG_HFP_AG
+	bool "[DEPRECATED] Bluetooth Hands Free Audio Gateway (HFP_AG) debug"
```

Comment:
Why are you flagging this as deprecated? Since BR/EDR is now officially maintained, I think the deprecation indications should likely be removed. What you might want to have however for new APIs is the experimental indication.

Reply:
After remove this configuration, no building issue is found, but the compliance check is reported https://github.com/zephyrproject-rtos/zephyr/actions/runs/8813249761/job/24190594128?pr=70532.

It seam the compliance check is not ready to support new logging.

Reply:
Just remove this Kconfig option completely